source/llassetgen/include/llassetgen/packing/internal/MaxRectsPacker.h

```diff
@@ -19,7 +19,7 @@ namespace llassetgen {
             bool pack(Rect<PackingSizeType>& rect);
 
            private:
-            LLASSETGEN_NO_EXPORT Rect<PackingSizeType>& findFreeRect(Rect<PackingSizeType>& rect);
+            LLASSETGEN_NO_EXPORT std::vector<Rect<PackingSizeType>>::iterator findFreeRect(Rect<PackingSizeType>& rect);
```

Comment:
Is the value referenced by the iterator actually written somewhere or could it be read-only / const?

Reply:
Can be const, thanks! Fixed in the latest commit.